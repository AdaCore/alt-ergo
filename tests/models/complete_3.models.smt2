(set-option :produce-models true)
(set-logic ALL)
(declare-const x Int)
(push 1)
(declare-fun f (Int) Int)
(assert (= (f 0) 1))
(pop 1)
; The function `f` isn't declared anymore in the current context and we must
; not produce a model value for it.
(check-sat)
(get-model)
