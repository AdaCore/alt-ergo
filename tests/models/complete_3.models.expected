
unknown
(
  (define-fun x () Int (as @a0 Int))
)
